nginx-controller/controller/controller.go

```diff
@@ -349,6 +379,26 @@ func (lbc *LoadBalancerController) syncEndp(task Task) {
 			if !lbc.isNginxIngress(&ing) {
 				continue
 			}
+			if ing.Annotations["nginx.org/mergible-ingress-type"] == "minion" {
+				master, err := lbc.findMasterForMinion(ing)
+				if err != nil {
+					glog.Errorf("Ignoring Ingress %v(Minion): %v", ing.Name, err)
+					continue
+				}
```

Comment:
let's run this check `lbc.cnf.HasIngress(&ing)` for the master

Reply:
Done.